0.2/Dockerfile

```diff
@@ -39,7 +39,7 @@ RUN apt update -y && apt install -y gpg unzip curl && \
 COPY conf/janusgraph-berkeleyje-lucene-server.properties conf/log4j-server.properties ${JANUS_HOME}/conf/gremlin-server/
 COPY scripts/remote-connect.groovy ${JANUS_HOME}/scripts/
 
-FROM openjdk:8-jre-slim-buster
+FROM eclipse-temurin:8-jdk
```

Comment:
Reason for this change: openjdk docker image is deprecated. 
See https://hub.docker.com/_/openjdk